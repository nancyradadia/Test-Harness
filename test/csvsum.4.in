1.51, 4.53
4.60, 8.90