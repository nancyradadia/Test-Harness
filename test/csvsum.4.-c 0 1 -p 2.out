Column 0 sum: 6.11
Column 1 sum: 13.43